util/libcrypto.num

```diff
@@ -4231,3 +4231,31 @@ X509_VERIFY_PARAM_get_time              4181	1_1_0d	EXIST::FUNCTION:
 EVP_PKEY_get0_poly1305                  4182	1_1_1	EXIST::FUNCTION:POLY1305
 DH_check_params                         4183	1_1_0d	EXIST::FUNCTION:DH
 EVP_PKEY_get0_siphash                   4184	1_1_1	EXIST::FUNCTION:SIPHASH
+ARIA_set_decrypt_key                    4185	1_1_1	EXIST::FUNCTION:ARIA
```

Comment:
No, no and no. No `ARIA_*` symbols should be exported from libcrypto shared library. Cipher should be accessible only through EVP. Just like every new algorithm. This does mean that aria.h should not be in include/openssl, that aria_{cbc|cfb|ctr|ecb|ofb}.c are essentially redundant, because corresponding functionality can be implemented directly in e_aria.c. [And it also means that there is no need for apps/speed.c modifications (speed -evp should be used)].